packages/hub/src/relative-values/components/views/ListView/sidebar.tsx

```diff
@@ -33,46 +33,92 @@ const TableRow: React.FC<TableRowProps> = ({ label, number }) => (
 let buildurl = (
   model: ModelEvaluator,
   numeratorItem: Item,
-  denominatorItem: Item
+  denominatorItem: Item,
+  variableName: string
 ) => {
   const toVarName = (id: string) => id.replace(/-/g, "_");
   const numeratorItemName = toVarName(numeratorItem.id);
   const denominatorItemName = toVarName(denominatorItem.id);
   return `${model.modelCode}
 // ------- AUTOGENERATED CODE -------
-dists = fn("${numeratorItem.id}", "${denominatorItem.id}")
-value_${numeratorItemName} = dists[0]
-value_${denominatorItemName} = dists[1]
-relativeValue = value_${numeratorItemName} / value_${denominatorItemName}
-median = abs(inv(dists[1], 0.5))
+dists = ${variableName}("${numeratorItem.id}", "${denominatorItem.id}")
+value_${numeratorItemName} = Dist(dists[0])
+value_${denominatorItemName} = Dist(dists[1])
+valueRatio = value_${numeratorItemName} / value_${denominatorItemName}
+medianDenominator = abs(median(value_${denominatorItemName}))
 tickFormat = '10'
 tickFormatObj = {tickFormat: tickFormat}
 
-[Plot.scatter({xDist:dists[1] / median, yDist: dists[0]/median, xScale: Scale.symlog(tickFormatObj), yScale: Scale.symlog(tickFormatObj)}), Plot.dist({
-  dist: relativeValue,
-  xScale: Scale.symlog(tickFormatObj),
-  yScale: Scale.symlog(tickFormatObj),
-  showSummary: false,
-})]`;
+@name("Normalized Scatter Plot")
+scatter = Plot.scatter(
+  {
+    xDist: SampleSet(dists[1] / medianDenominator),
+    yDist: SampleSet(dists[0] / medianDenominator),
+    xScale: Scale.symlog(
+      { tickFormatObj, title: "${numeratorItem.name}" }
+    ),
+    yScale: Scale.symlog(
+      { tickFormatObj, title: "${denominatorItem.name}" }
+    ),
+  }
+)
+
+@name("Debug")
+debug = [
+  scatter,
+  Tag.name(
+    [
+      Tag.name(
+        Plot.dist(dists[0], { xScale: Scale.symlog() }),
+        "${numeratorItem.name}"
+      ),
+      Tag.name(
+        Plot.dist(dists[1], { xScale: Scale.symlog() }),
+        "${denominatorItem.name}"
+      ),
+    ],
+    "Distributions"
+  ),
+]
+
+@name("Ratio Distribution")
+ratio = Plot.dist(
+  {
+    dist: valueRatio,
+    xScale: Scale.symlog(tickFormatObj),
+    yScale: Scale.symlog(tickFormatObj),
+    showSummary: false,
+  }
+)
+
+ratio
```

Comment:
I'm surprised you've decided to hide `scatter` plot by default; with negative*negative pairs the dist result might be confusing.

Reply:
Yea... I think this plot makes things more confusing for most users, at this point. I'll later add ``Tag.beginClosed``, which we'd use for ``debug`` so people can optionally open it.

I could also see having it open automatically when there are negative values.